localenv/mock-account-servicing-entity/app/routes/accounts.$accountId.tsx

```diff
@@ -23,7 +23,6 @@ export default function Transactions() {
           <th>Date</th>
           <th>Type</th>
           <th>Amount</th>
-          <th>Description</th>
```

Comment:
I considered adding a metadata column here since it is replacing description but wasn't sure how important that would be, and wasn't sure how to handle styling in light of it being fairly large (or even very large, potentially). Just figured it wasn't that important but I can add it if we want.

Reply:
I suggest 
```html
<td><code>{JSON.stringify(trx.metadata)}</code></td>


```

<img width="1512" alt="Screenshot 2023-06-28 at 18 59 29" src="https://github.com/interledger/rafiki/assets/15069181/f96cf085-ea13-4764-ae98-6cbceb6bbddc">

Reply:
added